cpovc_ovc/views.py

```diff
@@ -419,9 +419,9 @@ def ovc_view(request, id):
                        'care_giver' :care_giver,
                        'services': services, 'allow_edit': allow_edit,
                        'suppression': vl_sup,
-                       'cpara_count': 0,
-                       'case_plan_count': 0,
-                       'well_being_count': 0
+                       'cpara_count': services,

```

Comment:
This can be improved by passing just the dict value than the whole dict for each variable.